Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "a2"
-version = "0.7.0"
+version = "0.8.0"
```

Comment:
```suggestion
version = "0.7.0"
```

I'll bump the version in a separate PR as there is another open

Reply:
How about split the difference and I submit this as `0.7.1`?

Reply:
Sounds good to me! Thanks as always